.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```

Comment:
Remove this file from the commit, these are your local editor settings.

Reply:
Oh yes, I didn't realize this. Thanks a lot for pointing it out. I've deleted this file.